docs/resources/Audit_Log.md

```diff
@@ -128,6 +128,13 @@ If no object is noted, there won't be a `changes` array in the entry, though oth
 | AUTO_MODERATION_BLOCK_MESSAGE               | 143   | Message was blocked by Auto Moderation                    |                                                                                                                                                  |
 | AUTO_MODERATION_FLAG_TO_CHANNEL             | 144   | Message was flagged by Auto Moderation                    |                                                                                                                                                  |
 | AUTO_MODERATION_USER_COMMUNICATION_DISABLED | 145   | Member was timed out by Auto Moderation                   |                                                                                                                                                  |
+| CREATOR_MONETIZATION_REQUEST_CREATED        | 150   | Creator monetization request was created                  |                                                                                                                                                  |
+| CREATOR_MONETIZATION_TERMS_ACCEPTED         | 151   | Creator monetization terms were accepted                  |                                                                                                                                                  |
+| ROLE_PROMPT_CREATE                          | 160   | Role prompt was created                                   |                                                                                                                                                  |
+| ROLE_PROMPT_UPDATE                          | 161   | Role prompt was updated                                   |                                                                                                                                                  |
+| ROLE_PROMPT_DELETE                          | 162   | Role prompt was deleted                                   |                                                                                                                                                  |
+| GUILD_FEATURE_ITEM_CREATE                   | 171   | Guild feature item was created                            |                                                                                                                                                  |
+| GUILD_FEATURE_ITEM_UPDATE                   | 172   | Guild feature item was updated                            |                                                                                                                                                  |
```

Comment:
```suggestion
```

[Server home is being phased out](https://support.discord.com/hc/en-us/articles/6156116949911-Server-Home-Beta), so we shouldn't document these.

Reply:
docs: remove server home